isis/src/base/objs/Histogram/Histogram.cpp

```diff
@@ -138,9 +137,6 @@ namespace Isis {
 
     //set the ranges
     rangesFromNet(net,statFunc);
-
-    //add all the data to the now setup histogram
-    addMeasureDataFromNet(net,statFunc);
```

Comment:
Why was this removed? This looks like it doesn't add the data to the histogram anymore.